src/locales/messages.ts

```diff
@@ -887,6 +1027,11 @@ export default defineMessages({
     description: 'total cost is the sum of the infrastructure cost and supplementary cost',
     id: 'DashboardTotalCostTooltip',
   },
+  Description: {
+    defaultMessage: 'EN description',
+    description: 'description',
+    id: 'Description',
+  },
```

Comment:
This is a dup of CostModelsTagRateTableDesc.

Thinking CostModelsTagRateTableDesc should be removed in favor of this one.